app/api/v2/common/responses.py

```diff
@@ -33,7 +33,7 @@ def success(
     return json.ORJSONResponse(data, status_code, headers)
 
 
-class ErrorResponse(GenericModel, Generic[T]):
+class ErrorResponse(BaseModel, Generic[T]):
     status: Literal["error"]
     error: T
```

Comment:
separate thing but I think this shouldn't even be Generic[T] but rather just ServiceError